Assets/Scripts/PlayerCharacter.cs

```diff
@@ -2,14 +2,21 @@
 
 namespace FunMath
 {
-    public class PlayerCharacter : BaseCharacter
+    public class PlayerCharacter : MonoBehaviour
     {
         [SerializeField]
         [Range(0.1f, 2f)]
         float speed = 1f;
 
+        protected Rigidbody2D rb;
```

Comment:
We should use a better name here. Maybe full name to avoid confusion.

Reply:
Will rename in the next PR when I do a major rehaul.